README.rst

```diff
@@ -248,20 +253,47 @@ To simplify data conversion from Spark to TensorFlow, you can use Spark converte
     # make a tensorflow dataset from `converter1
     with converter1.make_tf_dataset() as dataset:
```

Comment:
There is only one DF and one converter in this example. Let's just call them df and converter.